Booking.java

```diff
@@ -56,10 +55,33 @@ public Review getReview() {
 
     @Override
     public boolean checkConstraints() {
-        return true;
+        if(constraint_13()) return true;
```

Comment:
if method constraint_13 is boolean you can simply do:
return constraint_13() but its readabilty only

Reply:
I didn't do it because there might be more constraints added to this function by you guys so all should be with IF statements